test/fixtures.rb

```diff
@@ -255,3 +255,12 @@ class PureValidator
 
   validates :name, presence: true
 end
+
+class UserDefinedValidationTest
+  include Lotus::Validations
+  attr_accessor :foo
+
+  validates :foo, :is_bar do |attribute|
```

Comment:
I agree. But in my opinion a validator should handle adding the validation error by itself:

```ruby
class BarValidator
  def self.call(attribute, errors)
    errors.add(attribute.name, Error.new(@name, name, result, @value)) unless attribute.value == 'bar'
   end
end

validates :foo, with: BarValidator 
```

Maybe there could be a class for validators which provide a an interface to simplify adding errors?